test/dynamo/test_higher_order_ops.py

```diff
@@ -36,6 +37,15 @@ def normalize_gm(gm_str):
     return remove_trailing_space(strip_comment(gm_str))
 
 
+def check_dynamic_shape_capture():
```

Comment:
What would be the correct way to know if any dynamic shape capturing is on?

Below in tests, we check the captured graph, which is slightly different with dynamic shapes on. So we want to skip this graph check for dynamic shapes case.

cc: @ezyang